toolkit/tools/internal/sliceutils/sliceutils.go

```diff
@@ -26,3 +31,8 @@ func FindMatches(slice []string, isMatch func(string) bool) []string {
 	}
 	return result
 }
+
+// StringMatch is intended to be used with "Find" for slices of strings.
+func StringMatch(expected, given interface{}) bool {
```

Comment:
I am of the opinion
```suggestion
// DoStringsMatch is intended to be used with "Find" for slices of strings.
func DoStringsMatch(expected, given interface{}) bool {
```

Reply:
In general for functions I'm also in favour of using names that describe an action `doX`, but in this case I'm using a noun "match" on purpose to describe the type of match between the elements we're interested in for a particular call to `sliceutils.Find`. It's the same idea as the `graphpath.NullHeuristic` elsewhere in my changes (that ones' part of an external library, though).